backend/src/Manager/Notification/NotificationLocalManager.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Manager\Notification;
+
+use App\AutoMapping;
+use App\Entity\NotificationLocalEntity;
+use App\Repository\NotificationLocalEntityRepository;
+use App\Request\Notification\NotificationLocalCreateRequest;
+use Doctrine\ORM\EntityManagerInterface;
+use DateTime;
+
+class NotificationLocalManager
+{
+    public function __construct(private AutoMapping $autoMapping, private EntityManagerInterface $entityManager, private NotificationLocalEntityRepository $notificationLocalRepository)
```

Comment:
We need to define the class properties outside the constructor function and initialize them inside the constructor body.